_includes/neurips/2019/about.neurips.2019.html

```diff
@@ -19,13 +19,13 @@ <h4>Creating Opportunity for Latinx in AI</h4>
         <div class="contents">
           <h3>Important Dates</h3>
             <ul>
-              <li>September 13th, 2019 (extended to Sep 20th): <a target='_blank' href="https://cmt3.research.microsoft.com/LXAINeurIPS2019">Abstract submission</a> deadline</li>
-              <li>September 9th, 2019: <a target='_blank' href="https://forms.gle/orfBj15ehMyAgPyJA">Program Committee application</a> deadline</li>
-              <li>September 27th, 2019: Abstract Review deadline</li>
-              <li>September 30th, 2019: Notification of submission acceptances</li>
-              <li>October 3rd, 2019: <a target='_blank' href="https://docs.google.com/forms/d/e/1FAIpQLSeMxJE96MyQ9KLFVX2HlB7TqLTCqhzTU_eUpz8pOwd6h8dh5g/viewform?usp=sf_link">LXAI Travel grant application</a> deadline</li>
-              <li>October 3rd, 2019: <a target='_blank' href="https://nips.cc/TravelApplication">NeurIPS Travel grant application</a> deadline</li>
-              <li>October 4th, 2019: Recommended deadline for <a target='_blank' href="http://www.cic.gc.ca/english/information/applications/visa.asp">Visa application</a></li>
+              <s><li>September 13th, 2019 (extended to Sep 20th): <a target='_blank' href="https://cmt3.research.microsoft.com/LXAINeurIPS2019">Abstract submission</a> deadline</li></s>
+              <s><li>September 9th, 2019: <a target='_blank' href="https://forms.gle/orfBj15ehMyAgPyJA">Program Committee application</a> deadline</li></s>
+		      <s><li>September 27th, 2019: Abstract Review deadline</li></s>
```

Comment:
Please try to keep lines indented to follow clean code principles for better legibility for others who will work on this site later.